shell/browser/api/electron_api_web_contents.cc

```diff
@@ -1812,6 +1812,81 @@ void WebContents::OnFirstNonEmptyLayout(
   }
 }
 
+// This object wraps the InvokeCallback so that if it gets GC'd by V8, we can
+// still call the callback and send an error. Not doing so causes a Mojo DCHECK,
+// since Mojo requires callbacks to be called before they are destroyed.
+class ReplySender : public gin::Wrappable<ReplySender> {
```

Comment:
`Replier`, `Replyable`, `ReplyChannel` 🤔 

I don't like the use of "sender" here, kinda confusing at a glance with the other concept of "sender"

Reply:
Ah, I didn't make the connection between "sender" as in "this is a thing that sends a reply" and "sender" as in `event.sender`. I'll change the name.